chord_drs/models.py

```diff
@@ -86,3 +91,14 @@ def __init__(self, *args, **kwargs):
         self.checksum = hash_obj.hexdigest()
 
         super().__init__(*args, **kwargs)
+
+    def return_minio_object(self):
+        if 's3' in self.location:
```

Comment:
this could get false positives - may want to use urlparse and check scheme

Reply:
Good observation, I've fixed it. Also included some stylistic fixes as per your comments.